display_information/control-annotation-sublayer-visibility/src/main/resources/control_annotation_sublayer_visibility/style.css

```
@@ -0,0 +1,7 @@
+.panel-region .label {
+  -fx-text-fill: white;
+}
+
+.panel-region .check-box {
```

This only really applies to `closedSublayerCheckbox` right now, so I think we could specify by writing:
```suggestion
#closedSublayerCheckbox {
```
Of course this depends on whether this works ok with the SV, so best test it out!

---

This works in the SV!